tool/autoqa/schemaorg/manual-annotations.js

```diff
@@ -563,12 +563,12 @@ const STRUCT_INCLUDE_THING_PROPERTIES = new Set([
 
 
 const STRING_FILE_OVERRIDES = {
-    'org.schema.Restaurant:Restaurant_name': 'org.openstreetmap:restaurant',
+    // 'org.schema.Restaurant:Restaurant_name': 'org.openstreetmap:restaurant',
     'org.schema.Person:Person_name': 'tt:person_full_name',
     'org.schema.Person:Person_alumniOf': 'tt:university_names',
     'org.schema.Person:Person_worksFor': 'tt:company_name',
     'org.schema.Person:Person_jobTitle': 'tt:job_title',
-    'org.schema.Hotel:Hotel_name': 'org.openstreetmap:hotel',
+    // 'org.schema.Hotel:Hotel_name': 'org.openstreetmap:hotel',
```

Comment:
What's special about Restaurant and Hotel? What about all the other domains? Your scripts should remove `#[string_values]` or replace them with a localized file instead.